samples/Grafana/README.md

```diff
@@ -0,0 +1,61 @@
+### Was this documentation helpful? [Share feedback](https://www.research.net/r/DGDQWXH?src=documentation%2Fgrafana)
+
+# Configure `dotnet monitor` in an AKS cluster for Managed Grafana
```

Comment:
Just to clarify my previous comment, I'm not opposed to putting the dashboard into source control here. I just don't want to put it in the samples folder since that suggests we can freely change it.